test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -341,7 +348,7 @@ TEST_F(StatisticsFromXMLProfileTests, XMLConfigurationForStatisticsDataWritersQo
     // it is just defined as data_writer profile. Thus, should not be created
     std::string subscription_througput_name = SUBSCRIPTION_THROUGHPUT_TOPIC;
     eprosima::fastdds::dds::DataWriter* subscription_througput_writer =
-            statistics_publisher_impl->lookup_datawriter(subscription_througput_name);
+            statistics_publisher->lookup_datawriter(subscription_througput_name);
```

Comment:
I know it is not related to these changes, but I suggest fixing this typo:

```suggestion
            statistics_publisher->lookup_datawriter(subscription_throughput_name);
```

Reply:
I think this previous suggestion was overlooked :sweat_smile: